swap-protocol-home.js

```diff
@@ -195,7 +199,8 @@ const setRewardApyDisplayValue = async (pools, siteData, cssIds) => {
   const swpPoolData = siteData['swpPoolData'];
   const swpRewardsPerYearByPool = siteData['swpRewardsPerYearByPool'];
 
-  for (const pool of pools) {
+
+  for (const pool in swpPoolData) {
```

Comment:
Since `pools` now has hyphens, we need to loop over the keys of the `swpPoolData` object, which uses colons.